docs/updating.md

```diff
@@ -951,7 +951,7 @@ pages for more details on the input and output variable types.
 #### plantcv.visualize.colorspaces
 
 * pre v3.8: NA
-* post v3.8: plotting_img = pcv.visualize.colorspaces(rgb_img)
+* post v3.8: plotting_img = pcv.visualize.colorspaces(rgb_img, original_img)
```

Comment:
@nfahlgren I thought I had done that here

Reply:
That's just the keyword though, since it has a default value it should be `original_img=True`

Reply:
Yep!